mega-zep-backend/src/main/java/com/gepardec/mega/domain/model/ProjectState.java

```diff
@@ -0,0 +1,18 @@
+package com.gepardec.mega.domain.model;
+
+public enum ProjectState {
+    OPEN,
+    WORK_IN_PROGRESS,
+    DONE,
+    NOT_RELEVANT;
+
+    public static ProjectState byName(String name) {
+        for (ProjectState projectState : ProjectState.values()) {
```

Comment:
Arrays.stream(ProjectState.values()).
                filter(v -> v.getTypeName().equalsIgnoreCase(typeName)).
                findFirst().
                orElseThrow(new IllegalArgumentException(String.format("No project state for given name '%s' present.", name)));